symengine/printers/unicode.h

```diff
@@ -0,0 +1,81 @@
+#ifndef SYMENGINE_UNICODE_H
+#define SYMENGINE_UNICODE_H
+
+#include <symengine/printers/strprinter.h>
+#include <symengine/printers/stringbox.h>
+
+namespace SymEngine
+{
+
+std::string print_double(double d);
+std::vector<std::string> init_str_printer_names();
+std::vector<unsigned> init_unicode_printer_lengths();
```

Comment:
does this need to be public, or can it be made a static function in unicode.cpp?

Reply:
It shouldn't be public. I'll remove the header declaration.